app/navigation/AppNavigation.tsx

```diff
@@ -402,31 +407,328 @@ export default ():any => {
       console.log("error")
     }
   }
- 
-  
+
+
   useEffect(() => {
-    if( userIsOnboarded == true){
-    createLocalNotificationListeners();
+    if (userIsOnboarded == true) {
+      createLocalNotificationListeners();
     }
   }, [userIsOnboarded]);
-  useEffect(() => {
+  const redirectPayload = (remoteMessage: any): any => {
 
-    const unsubscribe = messaging().onMessage(async remoteMessage => {
+    //console.log(remoteMessage, userIsOnboarded, navigationRef, "..redirectPayload")
+    if (remoteMessage && remoteMessage.data && remoteMessage.data.type && trimWhiteSpacePayload(remoteMessage.data.type) === "articles") {
```

Comment:
Can remoteMessage.data.type be an upper case? Are we not handling it if text comes other then lowercase. We should document it in the admin's manual if that's the case.